tests/console/system_prepare.pm

```diff
@@ -23,6 +24,18 @@ sub run {
 
     ensure_serialdev_permissions;
 
+    # Configure serial consoles for virtio support
+    # poo#18860 Enable console on hvc0 on SLES < 12-SP2
+    # poo#44699 Enable console on hvc1 to fix login issues on ppc64le
+    if (!check_var('VIRTIO_CONSOLE', 0)) {
+        if (is_sle('<12-SP2')) {
+            add_serial_console('hvc0');
```

Comment:
I don't think this will work, do we have that device available on all backends and architectures or only ppc64le?

Reply:
IIRC the virtio serial driver always uses the "HVC" serial terminal driver. So it will always get a device called `hvcN` regardless of the platform.

If not then we need to introduce another variable, but I don't think it is necessary for now.

Reply:
@okurz only sligty misleading PR name